datafusion/core/tests/sqllogictests/test_files/array.slt

```diff
@@ -149,7 +149,7 @@ select column1, column2, column3 from arrays_values_without_nulls;
 ### Array function tests
 
 
-## make_array
+## make_array (aliases: `make_list`)
```

Comment:
Any reason to have the alias `make_list`, may it be confused with creating ListArray?